dev-docs/bidders/nexx360.md

```diff
@@ -18,5 +18,8 @@ glv_id: 965
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description           | Example                              | Type      |
 |---------------|----------|-----------------------|--------------------------------------|-----------|
-| `account`     | required | Mediasquare owner ID  | `'1067'`                            | `string`  |
-| `tagId`       | required | Mediasquare code ID   | `'luvxjvgn'`                        | `string`  |
+| `account`     | required | Nexx360 account ID    | `'1067'`                             | `string`  |
+| `tagId`       | required | Nexx360 tag ID        | `'luvxjvgn'`                         | `string`  |
+| `bidfloor`       | optional | Bidfloor applied to auction (default: 0)        | `0.8`                         | `float`  |
+| `bidfloorCurrency`       | optional | Bidfloor currency (default: `USD`) - Can be `USD` or `EUR`       | `USD`                         | `string`  |
```

Comment:
USD doesn't look to be in string format in this example. Assuming it needs the wrapping ticks?